### ISSUE TITLE
Fastfetch not working properly

### ISSUE THREAD

**Issue by erlonbie** [kovidgoyal/kitty#7653]
**Describe the bug**
Fastfetch does not work properly inside kitty. It works as expected inside other terminal emulators.

**To Reproduce**
1. Install latest kitty through brew
2. Run fastfetch
3. See error

**Screenshots**
Kitty is the bigger, iTerm2 the top right and default terminal on bottom right.

![image](https://github.com/user-attachments/assets/a30ab357-f46f-4adf-b70f-0250d84f092a)

**Environment details**
```
kitty 0.35.2 (ad328bfeaa) created by Kovid Goyal
Darwin 192.168.1.18 23.4.0 Darwin Kernel Version 23.4.0: Fri Mar 15 00:11:05 PDT 2024; root:xnu-10063.101.17~1/RELEASE_X86_64 x86_64
ProductName:		macOS ProductVersion:		14.4.1 BuildVersion:		23E224
OpenGL: '4.1 INTEL-22.5.10' Detected version: 4.1
Frozen: True
Paths:
  kitty: /Applications/kitty.app/Contents/MacOS/kitty
  base dir: /Applications/kitty.app/Contents/Resources/kitty
  extensions dir: /Applications/kitty.app/Contents/Resources/Python/lib/kitty-extensions
  system shell: /bin/zsh
Loaded config files:
  /Users/erlonbie/.config/kitty/kitty.conf

Config options different from defaults:
active_tab_font_style      (True, False)
clipboard_control          ('write-clipboard', 'write-primary')
confirm_os_window_close    0
dim_opacity                0.75
disable_ligatures          1
dynamic_background_opacity True
editor                     /usr/local/bin/nvim
enable_audio_bell          False
enabled_layouts            ['grid']
font_family                MonoLisa Nerd Font Mono
font_size                  15.0
macos_option_as_alt        3
modify_font:
    cell_height 1
    cell_width 0
mouse_hide_wait            3.0
scrollback_lines           5000
symbol_map:
	U+e0a0 - U+e0a3 → PowerlineSymbols
	U+e0c0 - U+e0c7 → PowerlineSymbols
tab_title_template         {title}
touch_scroll_multiplier    5.0
url_prefixes               ('http', 'https', 'file', 'ftp', 'gemini', 'irc', 'gopher', 'mailto', 'news', 'git')
window_margin_width        FloatEdges(left=1.0, top=1.0, right=1.0, bottom=1.0)
window_padding_width       FloatEdges(left=1.0, top=1.0, right=1.0, bottom=1.0)
Added shortcuts:
	ctrl+f →  launch --location=hsplit --allow-remote-control kitty +kitten search.py @active-kitty-window-id
Removed shortcuts:
	cmd+, →  edit_config_file
	cmd+enter →  new_window
	cmd+h →  hide_macos_app
	cmd+k →  clear_terminal to_cursor active
	cmd+m →  minimize_macos_window
	cmd+n →  new_os_window
	cmd+q →  quit
	cmd+t →  new_tab
	cmd+w →  close_tab
	ctrl+cmd+, →  load_config_file
	ctrl+cmd+f →  toggle_fullscreen
	opt+cmd+h →  hide_macos_other_apps
	opt+cmd+r →  clear_terminal reset active
	opt+cmd+s →  toggle_macos_secure_keyboard_entry
	shift+cmd+/ →  open_url https://sw.kovidgoyal.net/kitty/
	shift+cmd+[ →  previous_tab
	shift+cmd+] →  next_tab
	shift+cmd+d →  close_window
	shift+cmd+w →  close_os_window
Changed shortcuts:
	kitty_mod+down →  scroll_page_down
	kitty_mod+e →  kitten hints
	kitty_mod+enter →  launch --cwd=current
	kitty_mod+l →  combine : clear_terminal scroll active : send_text normal,application \x0c
	kitty_mod+n →  new_os_window_with_cwd
	kitty_mod+t →  new_tab_with_cwd
	kitty_mod+up →  scroll_page_up
Colors:
	active_tab_background      #614a82   
	active_tab_foreground      #ddd8bb   
	background                 #1f1f28   
	color0                     #1f1f28   
	color1                     #e46a78   
	color10                    #9ec967   
	color11                    #f1c982   
	color12                    #7bc2df   
	color13                    #a98fd2   
	color14                    #7bc2df   
	color15                    #a8a48d   
	color2                     #98bc6d   
	color3                     #e5c283   
	color4                     #7eb3c9   
	color5                     #957fb8   
	color6                     #7eb3c9   
	color7                     #ddd8bb   
	color8                     #3c3c51   
	color9                     #ec818c   
	cursor                     #e6e0c2   
	cursor_text_color          #1f1f28   
	foreground                 #ddd8bb   
	inactive_tab_background    #363644   
	inactive_tab_foreground    #ddd8bb   
	selection_background       #49473e   
	selection_foreground       #ddd8bb   

Important environment variables seen by the kitty process:
	PATH                                /Applications/kitty.app/Contents/MacOS:/usr/bin:/bin:/usr/sbin:/sbin
	LANG                                en_US.UTF-8
	SHELL                               /bin/zsh
	USER                                erlonbie

```

**Additional context**
Using `kitty --config NONE` the problem goes away, probably something wrong with my config, but I don't know waht.

**Comment by kovidgoyal** [comment]
So report the issue to fastfetch.

**Comment by erlonbie** [comment]
I don't want to sound rude, but shouldn't this be treated in kitty? Since the problem does not occur in other terminal emulators

**Comment by erlonbie** [comment]
But thanks for the quick response, I'll report to fastfetch

**Comment by CarterLi** [comment]
You said `probably something wrong with my config` but you didn't upload your config file

Are you using Hackintosh?

**Comment by kovidgoyal** [comment]
There's no fastfetch specific facility in kitty. If I had to guess, you have changed the value of TERM in kitty.conf. But the developer of fastfetch is best placed to answer why it doesnt work.

**Comment by CarterLi** [comment]
Should have been fixed by https://github.com/fastfetch-cli/fastfetch/commit/7f2ab7310aab8c8f6e72ce3abf40db40f14cfa0e

**Comment by erlonbie** [comment]
> You said `probably something wrong with my config` but you didn't upload your config file
> 
> Are you using Hackintosh?

Yes

**Comment by erlonbie** [comment]
> Should have been fixed by [fastfetch-cli/fastfetch@7f2ab73](https://github.com/fastfetch-cli/fastfetch/commit/7f2ab7310aab8c8f6e72ce3abf40db40f14cfa0e)

It is indeed! Thank you
